maxim/logger/livekit/openai/realtime/handler.py

```
@@ -55,52 +57,54 @@ def handle_openai_client_event_queued(session_info: SessionStoreEntry, event: di
-        turn = session_info["current_turn"]
-        if turn["turn_input_audio_buffer"] is None:
-            turn["turn_input_audio_buffer"] = b""
+        turn = session_info.current_turn
+        if turn.turn_input_audio_buffer is None:
+            turn.turn_input_audio_buffer = BytesIO()
         decoded_audio = None
         try:
             decoded_audio = base64.b64decode(event["audio"])
         except Exception:
-            pass      
+            pass
```

Ruff (0.11.9)

Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

Replace with `contextlib.suppress(Exception)`

(SIM105)